### DISCUSSION TITLE
song request page is empty, table says "no records to display"

### DISCUSSION THREAD

**Discussion by valiant8086** [AzuraCast/AzuraCast#6248] [Q&A]
So I had to delete all songs from the sftp and reupload different ones. Song Requesting was displaying the songs prior to that but now when I hit request song the table does have options to select from 4 pages, but it says no records to display. I have just the default playlist and it has a little over 10,000 items in it, and they display when I go to the "number of songs " link displayed in the playlists table.

The station has internal player disabled for now, is using a remote streaming relay only. It insists the station is offline where the title of the currently playing song should be, but in the icecast remote server the currently playing song is identified in the now playing area.

Just throwing stuff out there because I don't know if any of this could be an indication.

Sftp seems to be pretty weak, I had to ditch Win SCP and just connect with rclone and had it set to run checkers first, do only one checker, and do only one transfer in order to get it to stop either erroring out or going really slowly. It's new that it's like that though, before sftp worked well with win scp and uploading into it from the same computer was going at about 40mb/s with no issue. Only after I deleted the entire contents the first time did it start being silly.

**Comment by BusterNeece** [comment]
I suspect this has been fixed due to our retirement of the Meilisearch engine in favor of optimizing our database queries elsewhere. If not, feel free to update and let us know what's going wrong with additional logs.
